cmd/embedded-cluster/restore.go

```diff
@@ -979,6 +960,11 @@ var restoreCommand = &cli.Command{
 			}
 		}
 
+		applier, err := getAddonsApplier(c, "")
```

Comment:
nit: should this be moved to the `ecRestoreStateNew` case since it's only used there?